tools/ddsrouter_tool/src/cpp/main.cpp

```diff
@@ -65,6 +68,13 @@ int main(
         return static_cast<int>(arg_parse_result);
     }
 
+    if (print_version)
```

Comment:
I would use this argument as another `ui::ProcessReturnCode` as `help`, so when called it does not execute the process. I know the actual structure is a bit mess as help is get and printed in `arguments_configuration.cpp` rather than here, but I would try to keep this new argument with the same behaviour of `help` and same structure.
Feel free to refactor it if you consider it necessary, but keep help and version parallel